src/App.js

```diff
@@ -43,9 +49,10 @@ class App extends Component {
     return (
       <main className='App'>
         <h1> Rotten Tomitillos </h1>
-        {this.state.error && <h2>{this.state.error}</h2>}
-        <AllMovies movies={this.state} showMore={this.showMore} />
-        {(this.state.movies.length === 1) && <SingleView movie={this.state.movies[0]} />}
+
+        {/* <AllMovies movies={this.state} showMore={this.showMore} /> */}
+        {(this.state.selected) ? <SingleView movie={this.state.selected} showAll={this.showAll} /> : <AllMovies movies={this.state} showMore={this.showMore} />}
```

Comment:
Some developers may find multi-line ternaries to be more difficult to read. Make sure you are also getting familiar with the logical-and (&&) syntax as an alternate option. Consider which YOU find more readable.